sdk/attestation/azure-security-attestation/cgmanifest.json

```diff
@@ -0,0 +1,36 @@
+{
+    "Registrations": [
+        {
+            "Component": {
+                "Type": "git",
+                "git": {
+                    "RepositoryUrl": "https://github.com/google/googletest",
+                    "CommitHash": "703bd9caab50b139428cea1aaff9974ebee5742e"
+                }
+            },
+            "DevelopmentDependency": true
+        },
+        {
+            "Component": {
+                "Type": "other",
+                "Other": {
+                    "Name": "clang-format",
+                    "Version": "9.0.0-2",
+                    "DownloadUrl": "https://ubuntu.pkgs.org/18.04/ubuntu-updates-universe-amd64/clang-format-9_9-2~ubuntu18.04.2_amd64.deb.html"
+                }
+            },
+            "DevelopmentDependency": true
+        },
+        {
+            "Component": {
+                "Type": "other",
+                "Other": {
+                    "Name": "doxygen",
+                    "Version": "1.8.20",
+                    "DownloadUrl": "http://doxygen.nl/files/doxygen-1.8.20-setup.exe"
```

Comment:
This is the same as in other cgmanfiest.json files around the repo... However, in order to prevent MITM attacks we've been downloading doxygen from our own storage account via HTTPS (this package was initially downloaded from SourceForge via HTTPS) 

https://github.com/Azure/azure-sdk-for-cpp/blob/c10de791cceeb393694e86c912267d5faa89bb94/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L84 

To that end, I think `DownloadUrl` should be `https://azuresdktooling.blob.core.windows.net/dependencies/doxygen-1.8.18-setup.exe` if we intend to accurately reflect where we fetch the file. 

I think @vhvb1989 worked on cgmanifest files... Is there a reason we use this http URL instead of the URL from which we're hosting the file?